schemas/com.snowplowanalytics.snowplow/web_tab/jsonschema/1-0-0

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a web tab context",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "web_tab",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": ["string", "null"],
```

Comment:
Question: I know we made this decision to be able to return null for the `tabId` in case we can't persist it in storage, just thinking that since now the ID has it's own context, wouldn't it be better to just not add the context entity in case we don't have the ID? The context entity without the ID doesn't make much sense to me.